lib/lita/handlers/github_pinger.rb

```diff
@@ -212,6 +222,14 @@ def send_dm(username, content)
           alert_eng_pr("Could not find user with name #{username}, please configure everbot.")
         end
       end
+
+      def detect_type(body)
+        if body['pull_request']
+          'pull_request'
+        elsif body['issue']
+          'issue'
```

Comment:
Let's get these strings a little more human-friendly (`Pull Request`, `Issue`), since they will be used in the slack message!